app/views/proposals/_form.html.erb

```diff
@@ -5,15 +5,15 @@
   <% unless user_signed_in? || current_or_guest_user.first_name != 'guest' %>
     <%= f.simple_fields_for :user do |h| %>
       <%= h.input :first_name,
-      label: "Let your friends know who you are",
+      label: false,
       placeholder: "Your name",
       input_html: {id: 'sign_up_code',
       value: nil } %>
     <% end %>
   <% end %>
-  <%= f.input :place, label: "Add a new place" %>
-  <%= f.association :tags, class: 'tag-selector', collection: current_or_guest_user.tags, label: "Add tags to this place (max 20 char)" %>
-  <%= f.button :submit, value: "Add a new place", class: 'btn btn-choogle btn-proposal-margin' %>
+  <%= f.input :place, label: false, placeholder: "Search a place", error: "Please add a place"%>
+  <%= f.association :tags, class: 'tag-selector', collection: current_or_guest_user.tags, label: "Describe this place with your own words" %>
```

Comment:
Ok, but the user has to get that we want **single** words. But let's try this anyway, it's a fine solution!

Reply:
Humm, your right I forget that they can understand this as a free text input with sentences. I propose "describe this place with tags". I think we should mention that tags are related to the specific place and not the event. The better solution would be to have tags-like formated placeholders.